src/blocks/card.css

```diff
@@ -0,0 +1,83 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  height: 134px;
+  width: 160px;
+  margin: 0 auto 40px;
+}
+
+.card__caption {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__product-code {
+  align-self: flex-start;
+  margin-bottom: 16px;
+
+  font-size: 10px;
+  line-height: 14px;
+  color: #616071;
+  font-weight: 400;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+  width: 100%;
```

Comment:
I don't think that you need to use 'width: 100%', the code should work without it

Reply:
fixed!